pgscm/templates/admin/macro.html

```diff
@@ -0,0 +1,20 @@
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% macro render_btn(id, size='btn-xs', type='btn-default', modal_target='modal', title="", css_class='plus', data=None) -%}
```

Comment:
if possible, please move to common macro.html file and try to reuse it as much as possible. I don't want to merge this kind of code.